README.md

```diff
@@ -13,3 +13,200 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+
+//
+
+
+Name: Kane Kriz
+
+Start Date: 11 Feb 2025
+
+Feedback Request 1: 12 Feb 2025
+
+
+//
+
+
+Feedback Request 2: 13 April 2025
+
+
+//
+
+
+
+
+(
+
+I added the bolded titles in to better separate the reasoning of the issues from one another as in the original feedback request they were kind of bleeding into one another and making it hard to follow my reasoning.
+Also to make it more evident that reason 2 is not the same as reason 1, which was one of the outlined issues from the edits requested.
+
+Thank you.
+
+)
+
+
+
+
+Response 1: 
+
+
+**Disregard for lower terms**
+
+The first issue with asymptotic analysis being potentially misleading is due to a fundamental part of asymptotic analysis as a whole.
+
+Asymptotic analysis focuses upon the highest order and most important term in regards to performance.
+
+However, smaller order terms still make an impact in realistic scenarios without notably large (asymptotic) input sizes.
+
+In varying situations, these smaller order terms still can make a difference to the overall performance and runtime, though with asymptotic analysis, they are intentionally ignored.
+
+
+
+
+
+
+**Oversimplified Algorithm groupings by highest order terms**
+
+Another potential issue with asymptotic analysis is that it groups algorithms only by their highest order terms.
+
+This differs from the first reason by focusing on how defined asymptotic runtimes hide practical differences between algorithms with shared asymptotic complexity, rather than just conceptually ignoring lower order terms.
```

Comment:
I still don't understand it. It seems to follow directly from the previous reason. I don't see how this is a different reason.